rules/opentitan.bzl

```diff
@@ -50,41 +50,6 @@ PER_DEVICE_DEPS = {
 def flatten(l):
     return [item for ll in l for item in ll]
 
-def _obj_transform_impl(ctx):
```

Comment:
Are there any uses of the remaining macro "rules" in this file (like `opentitan_binary`, `opentitan_rom_binary`, etc)?

They should be candidates for removal as well.

Reply:
nvm: just re-read your PR description.